clients/client-core/src/client/cover_traffic_stream.rs

```diff
@@ -142,6 +143,19 @@ impl LoopCoverTrafficStream<OsRng> {
         self.packet_size = packet_size;
     }
 
+    pub fn try_set_extended_packet_size(&mut self, size: &str) -> bool {
```

Comment:
would it be perhaps cleaner to pass an explicit enum here so that it couldn't fail?